src/comps/product-listing/productListing.jsx

```diff
@@ -3,15 +3,66 @@ import "./productListing.css"
 
 import { useProduct} from "./../../context/products.context.jsx"
 
+import { useFilter } from "../../context/filter.context.jsx"
 
 const ProductListing = ()  => {
 
-    const {itemList, setItemList} = useProduct()
+    const {itemList} = useProduct()
+    
+
+    const { state } = useFilter()
+
+    const priceFilter = (products, priceSort) =>
+    {
+        if(priceSort === "low_to_high")
+        {
+          return( products.sort((a,b) => a.price - b.price)
+          )
```

Comment:
because we have single statement after if we can do remove {} braces, also you can remove these ()  after return statement